monitoring/tools/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.2"
```

Comment:
This is fine for now, but I think we should probably move to release-based versioning.